### ISSUE TITLE
Bug 54109031: [React Native Gallery -> Radio Buttons]: Tab navigation is defined for radio buttons.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#497]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: Keyboard

Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘Radio Buttons’ button present under “Basic Input” group and invoke it.
Now navigate in between the radio buttons and observe the keyboard behaviour.
Actual Result:
Tab navigation is defined for radio buttons. While navigating using tab key, keyboard focus is landing on each radio button.
 
Expected Result:
Tab navigation should not be defined for radio buttons. User should be able to navigate in between the radio buttons using arrow keys only.
When user hit tab key from the hamburger button focus should land on the first radio button and from there arrow keys navigation should be defined for the radio buttons.
 
User Impact:
Users with motor disabilities who rely on keyboard will be impacted if tab navigation is defined for the radio buttons.

Link to ADO bug: https://dev.azure.com/microsoft/OS/_workitems/edit/54109031

**Comment by chrisglein** [comment]
Behavior doesn't match what a WinUI Gallery radio group does.
The community module radio buttons not having the same behavior isn't surprising.
Interestingly the RN-XAML version of radio buttons has the wrong behavior too.
